### DISCUSSION TITLE
5ghz network injection failure for channel 104 & 100

### DISCUSSION THREAD

**Discussion by ARTHURTITO-R2D2** [aircrack-ng/aircrack-ng#2626] [Q&A]
I ´m doing some injection testing on my AP for 5GHz with a MT7612U , for some reason the injection does not work for channel 104 or 100 but it does for others like 161 or 157, I tried changin the region  and  frequency  as maybe the spectrum is different , but it does not work, need some help or fresh ideas please

**Comment by ZeroChaos-** [comment]
Injection is not permitted by the kernel on channels which are DFS.  If you do `iw reg get` the channels marked "PASSIVE_SCAN" means you are not allowed to inject.
